deployment/docker/Makefile

```diff
@@ -102,24 +102,23 @@ run-via-docker: package-via-docker
 
 app-via-docker-goos-goarch:
 	APP_SUFFIX='-$(GOOS)-$(GOARCH)' \
-	DOCKER_OPTS='--env CGO_ENABLED=$(CGO_ENABLED) --env GOOS=$(GOOS) --env GOARCH=$(GOARCH)' \
+	DOCKER_OPTS='--env CGO_ENABLED=$(CGO_ENABLED) --env GOOS=$(GOOS) --env GOARCH=$(GOARCH) $(foreach v,$(EXTRA_ENVS),--env $(v))' \
 	$(MAKE) app-via-docker
 
 app-via-docker-pure:
 	APP_SUFFIX='-pure' DOCKER_OPTS='--env CGO_ENABLED=0' $(MAKE) app-via-docker
 
 app-via-docker-linux-amd64:
+	EXTRA_ENVS='CC=/opt/cross-builder/x86_64-linux-musl-cross/bin/x86_64-linux-musl-gcc' \
 	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 $(MAKE) app-via-docker-goos-goarch
 
 app-via-docker-linux-arm:
-	APP_SUFFIX='-linux-arm' \
-	DOCKER_OPTS='--env CGO_ENABLED=0 --env GOOS=linux --env GOARCH=arm --env GOARM=5' \
-	$(MAKE) app-via-docker
+	EXTRA_ENVS='GOARM=5' \
```

Comment:
I'm strongly against this change. v5 version of ARM platform is very primitive and not widely used. Also, it usually requires to build it with software float support, since it doesnt have FPU.

I think, we have to use ARMv8 (default empty value).

https://en.wikipedia.org/wiki/ARM_architecture_family#Cores

Reply:
didn't change actions that are happening in this target, just reusing common one, like for other platforms

Reply:
@f41gh7 --env GOARM=5 is a part of our code already, should we change it to something different?

Reply:
It looks like `GOARM=5` is still needed according to https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4965#issuecomment-2083398379

Reply:
See also https://github.com/golang/go/issues/62475